extension/public/background.ts

```diff
@@ -1,11 +1,27 @@
-import { initMessageListener } from "../src/background";
+import { initMessageListener } from "background";
+import { browser } from "webextension-polyfill-ts";
```

Comment:
This appears to be the best way to use the webextension polyfill with TS, instead of as a global. Having this imported should make Firefox support basically a find/replace `chrome.` -> `browser.`

Reply:
nice!